### DISCUSSION TITLE
Error boiling Instant-ngp

### DISCUSSION THREAD

**Discussion by brahimfarhat** [NVlabs/instant-ngp#1491] [General]
Hi, I'm facing the following error when trying to build the software: 

/usr/include/c++/11/bits/std_function.h:435:145: error: parameter packs not expanded with ‘...’:
  435 |         function(_Functor&& __f)
      |                                                                                                                                                 ^ 
/usr/include/c++/11/bits/std_function.h:435:145: note:         ‘_ArgTypes’
/usr/include/c++/11/bits/std_function.h:530:146: error: parameter packs not expanded with ‘...’:
  530 |         operator=(_Functor&& __f)
      |                                                                                                                                                  ^ 
/usr/include/c++/11/bits/std_function.h:530:146: note:         ‘_ArgTypes’
gmake[2]: *** [dependencies/tiny-cuda-nn/CMakeFiles/tiny-cuda-nn.dir/build.make:76: dependencies/tiny-cuda-nn/CMakeFiles/tiny-cuda-nn.dir/src/common_host.cu.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:394: dependencies/tiny-cuda-nn/CMakeFiles/tiny-cuda-nn.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2

Any help ?

Thank you!

**Comment by jcamachado** [comment]
Same Issue
